docs/4-querying-lists.md

```diff
@@ -17,6 +17,21 @@ const queryObservable = af.database.list('/items', {
 });
 ```
 
+**Query Options:**
+
+| method   | purpose            | 
+| ---------|--------------------| 
+| orderByChild | specify a child to order by |
+| orderByKey | boolean to order by Firebase keys |
```

Comment:
Anywhere you mention the database, make sure to use the term "Firebase Database" rather than just Firebase. The database is one of 14 features of Firebase, so it's good to be specific.